web/packages/teleport/src/Login/Login.tsx

```diff
@@ -78,3 +90,114 @@ export function LoginComponent({
     </>
   );
 }
+
+const LicenseBox = styled(Box)`
+  width: 100%;
+  max-width: 550px;
+  padding: ${props => props.theme.space[4]}px;
+  background-color: ${props => props.theme.colors.levels.surface};
+  border-radius: ${props => props.theme.radii[3]}px;
+  margin: auto;
+`;
+
+function LicenseAcknowledgement({
+  setLicenseAcknowledged,
+}: {
+  setLicenseAcknowledged(): void;
+}) {
+  const [checked, setChecked] = useState(false);
+
+  return (
+    <>
+      <LogoHero />
+      <LicenseBox>
+        <InfoHeader mb={2}>Welcome to Teleport</InfoHeader>
+        <InfoText>
+          Companies may use Teleport Community Edition on the condition they
+          have less than 100 employees and less than $10MM in annual revenue. If
+          your company exceeds these limits, please{' '}
+          <Text
+            as="a"
+            href="https://goteleport.com/signup/enterprise/?utm_campaign=CTA_terms_and_conditions&utm_source=oss&utm_medium=in-product"
+            target="_blank"
+          >
+            contact us
+          </Text>{' '}
+          to evaluate and use Teleport.
+        </InfoText>
+        <Flex mt={3} alignItems="center" gap={2}>
+          <StyledCheckbox
+            checked={checked}
+            onChange={e => {
+              setChecked(e.target.checked);
+            }}
+          />
+          By clicking continue, you agree to our{' '}
+          <Text
+            as="a"
+            href="https://github.com/gravitational/teleport/blob/master/LICENSE-community"
+            target="_blank"
+          >
+            Terms and Conditions
```

Comment:
nit: can we decrease the space to match the text spacing (between `our` and `Terms`), it's probably the `gap={2}` affecting it

<img width="243" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/9ef9f718-9cf6-4784-bf4b-311401c71137">

Reply:
FWIW, I don't think it needs to be a separate element. Please let's also remember to use labels for checkboxes, this way the user doesn't have to hunt for the small checkbox and instead can click on the text.

<details>
<summary>Patch</summary>

```diff
diff --git a/web/packages/teleport/src/Login/Login.tsx b/web/packages/teleport/src/Login/Login.tsx
index 985f78b812..5c51d1a1ba 100644
--- a/web/packages/teleport/src/Login/Login.tsx
+++ b/web/packages/teleport/src/Login/Login.tsx
@@ -19,7 +19,7 @@
 import React, { useState } from 'react';
 import styled from 'styled-components';
 
-import { Box, Text, Flex, ButtonPrimary } from 'design';
+import { Box, Text, Link, Flex, ButtonPrimary } from 'design';
 import { StyledCheckbox } from 'design/Checkbox';
 
 import FormLogin from 'teleport/components/FormLogin';
@@ -116,31 +116,30 @@ function LicenseAcknowledgement({
           Companies may use Teleport Community Edition on the condition they
           have less than 100 employees and less than $10MM in annual revenue. If
           your company exceeds these limits, please{' '}
-          <Text
-            as="a"
+          <Link
             href="https://goteleport.com/signup/enterprise/?utm_campaign=CTA_terms_and_conditions&utm_source=oss&utm_medium=in-product"
             target="_blank"
           >
             contact us
-          </Text>{' '}
+          </Link>{' '}
           to evaluate and use Teleport.
         </InfoText>
-        <Flex mt={3} alignItems="center">
+        <Flex as="label" mt={3} gap={2} alignItems="center">
           <StyledCheckbox
             checked={checked}
             onChange={e => {
               setChecked(e.target.checked);
             }}
           />
-          <Text ml={2}>By clicking continue, you agree to our </Text>
-          <Text
-            ml={1}
-            as="a"
-            href="https://github.com/gravitational/teleport/blob/master/LICENSE-community"
-            target="_blank"
-          >
-            {' '}
-            Terms and Conditions
+          <Text>
+            By clicking continue, you agree to our{' '}
+            <Link
+              href="https://github.com/gravitational/teleport/blob/master/LICENSE-community"
+              target="_blank"
+            >
+              Terms and Conditions
+            </Link>
+            .
           </Text>
         </Flex>
         <ButtonPrimary

```
</details>

<img width="627" alt="login" src="https://github.com/gravitational/teleport/assets/27113/5f984576-e856-45ae-9177-8195bd2ba59e">

Reply:
LOVE that, thank you